### PR TITLE
Created upload dag script for easier local test environment devops

### PR DESCRIPTION
**Pull request by Sethvg**
Adding script to help upload all dags to cloud composer.

### CODE REVIEW

**Review by ldbnr4 — APPROVED** [review]
Nice!

Do you think you can have the deployToTest github workflow use this script? If not, no biggie.
satcher/health-equity-tracker/.github/workflows/deployToTest.yml

File: airflow/upload-to-environment.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+for filename in dags/*; do
+  echo "Uploading Dag: ${filename}"
+    gcloud composer environments storage dags import --environment data-ingestion-environment --source $filename --location=us-central1
```
Comment (vanshkumar) [inline airflow/upload-to-environment.sh:4]: Does this location have to be explicitly specified?

**Review by vanshkumar — APPROVED** [review]

File: airflow/upload-to-environment.sh (hunk 1 above)
Comment (ldbnr4) [reply airflow/upload-to-environment.sh:4]: Yeah it's a required flag.

https://cloud.google.com/sdk/gcloud/reference/composer/environments/storage/dags/import

**Comment by vanshkumar** [comment]
Hey Seth, are you planning to submit this (after updating the landing bucket & manual upload bucket)? I'm using the scripts in here locally, they're pretty helpful

File: build_docker_images.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+ingestion_image_name="ingestion-service"
```
Comment (vanshkumar) [inline build_docker_images.sh:1]: Ah, I actually already have a PR out to add basically this exact script here: https://github.com/SatcherInstitute/health-equity-tracker/pull/357

Do you want to just review that one and remove this file from this PR?

**Review by vanshkumar — APPROVED** [review]

File: build_docker_images.sh (hunk 2 above)
Comment (Sethvg) [reply build_docker_images.sh]: Removed

**Pull request merged**